data/emission_factors.csv

```diff
@@ -1,100 +1,307 @@
+211,VN,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.37789,
+212,EH,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.79231,
+213,YE,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.58073,
+214,ZM,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.06617,
+215,ZW,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.39607,
+216,global,public_transport,bus,mobitool,Stadtbus (9m),P.km,small,local,cng,kg/P.km,0.1464,
+217,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,medium,local,cng,kg/P.km,0.1034,
+218,global,public_transport,bus,mobitool,Stadtbus (18m),P.km,large,local,cng,kg/P.km,0.0709,
+219,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,average,local,cng,kg/P.km,0.1034,
+220,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,average,local,diesel,kg/P.km,0.1338,
+221,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,medium,local,diesel,kg/P.km,0.1338,
+222,global,public_transport,bus,mobitool,Stadtbus (18m),P.km,large,local,diesel,kg/P.km,0.0905,
+223,global,public_transport,bus,mobitool,Stadtbus (9m),P.km,small,local,diesel,kg/P.km,0.1716,
+224,global,public_transport,bus,mobitool,Reisebus,P.km,average,long-distance,diesel,kg/P.km,0.0465,
+225,global,public_transport,bus,mobitool,Trolleybus,P.km,average,local,electric,kg/P.km,0.0296,
+226,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,cng,kg/P.km,0.1528,
+227,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,cng,kg/P.km,0.1033,
+228,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,cng,kg/P.km,0.1233,
+229,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,average,kg/P.km,0.1864,
+230,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,gasoline,kg/P.km,0.186,
+231,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,gasoline,kg/P.km,0.1297,
+232,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,gasoline,kg/P.km,0.1506,
+233,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,gasoline,kg/P.km,0.2004,
+234,global,vehicle,car,mobitool,Personenkraftwagen,P.km,average,,diesel,kg/P.km,0.189,
+235,global,vehicle,car,mobitool,Personenkraftwagen,P.km,small,,diesel,kg/P.km,0.1109,
+236,global,vehicle,car,mobitool,Personenkraftwagen,P.km,medium,,diesel,kg/P.km,0.1322,
+237,global,vehicle,car,mobitool,Personenkraftwagen,P.km,large,,diesel,kg/P.km,0.1757,
+238,global,public_transport,train,mobitool,Eisenbahn,P.km,,long-distance,,kg/P.km,0.0068,
+239,global,public_transport,train,mobitool,Eisenbahn,P.km,,local,,kg/P.km,0.008,
+240,global,public_transport,train,mobitool,Eisenbahn,P.km,,average,,kg/P.km,0.007,
+241,global,public_transport,train,mobitool,Strassenbahn,P.km,,local,electric,kg/P.km,0.0428,

```

Comment:
Can we put "tram" as subcategory here? We are currently handling it as an extra transportation mode (and I think it makes sense, in case we add emission factors for a local electric train, we would not be able to differentiate other than via the name)

Reply:
done

Reply:
updated the readme, too